app/util/BreakingNewsClient.scala

```diff
@@ -0,0 +1,91 @@
+package util
+
+import com.gu.mobile.notifications.client.models.{BreakingNewsPayload, Topic}
+import logging.Logging
+import play.api.libs.json.{JsError, JsSuccess, Json, Reads}
+import play.api.libs.ws.{WSClient, WSResponse}
+import updates.ClientHydratedTrail
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
+
+case class BreakingNewsRequest(trail: ClientHydratedTrail, topic: Topic, payload: BreakingNewsPayload)
+
+case class BreakingNewsResponse(id: String)
+
+object BreakingNewsResponse {
+  implicit val jf = Json.format[BreakingNewsResponse]
+}
+
+sealed trait BreakingNewsError
+case class BreakingNewsResponseStatusError(status: Int, body: String) extends BreakingNewsError
+case class BreakingNewsResponseFormatError(error: Either[String, Throwable], body: String) extends BreakingNewsError
+case class BreakingNewsResponseException(err: Throwable) extends BreakingNewsError
+case class BreakingNewsSuppressedError(message: String) extends BreakingNewsError
+
+object BreakingNewsError {
+  def getDescription(error: BreakingNewsError): String = error match {
+    case BreakingNewsResponseStatusError(status, body) => s"Unexpected HTTP status $status: $body"
+    case BreakingNewsResponseFormatError(Left(description), body) => s"Format error in response $description: $body"
+    case BreakingNewsResponseFormatError(Right(err), body) => s"Format error in response: ${err.getMessage}: $body"
+    case BreakingNewsResponseException(err) => s"Unexpected error ${err.getMessage}"
+    case BreakingNewsSuppressedError(message) => message
+  }
+
+  def getThrowable(error: BreakingNewsError): Option[Throwable] = error match {
+    case BreakingNewsResponseFormatError(Right(err), _) => Some(err)
+    case BreakingNewsResponseException(err) => Some(err)
+    case _ => None
+  }
+}
+
+class BreakingNewsClient(
+  host: String,
+  apiKey: String,
+  ws: WSClient
+) extends Logging {
+
+  private val url = s"$host/push/topic"
+
+  def send(request: BreakingNewsRequest)(implicit ec: ExecutionContext): Future[Either[BreakingNewsError, BreakingNewsResponse]] = {
+    val json = Json.stringify(Json.toJson(request.payload))
+    val requestBody = json.getBytes("UTF-8")
+
+    post(url, apiKey, requestBody) map { response =>
+      if(response.status == 201) {
+        validateFormat[BreakingNewsResponse](response.body)
+      } else {
+        Left(BreakingNewsResponseStatusError(response.status, response.body))
+      }
+    } recover {
+      case NonFatal(e) => Left(BreakingNewsResponseException(e))
```

Comment:
I think here if you use
`case NonFatal(e: Exception) => Left(BreakingNewsResponseException(e))`

you can then use `Exception` everywhere else